resources/scripts/jquery/jquery-utils.js

```diff
@@ -308,6 +308,25 @@
     	$(".eXist_wizard_next", container).click(function () {
     		if (currentPage == pages.length - 1)
     			return;
```

Comment:
This return statement belongs to the previous row, the following lines should be at the same indentation level as the if-statement in line 309

Reply:
It was already there :) and all the `return` are indented that way throughout the whole file. Shall I change them all?

Reply:
Sorry for my misleading comment. The code you added is indented one level too much. That's all.

Reply:
Ah, OK, now I got it. Is it any better now?